www/src/html.js

```diff
@@ -63,6 +63,7 @@ export default class HTML extends React.Component {
           />
           {this.props.postBodyComponents}
           <link
+            rel="preload"
```

Comment:
This won't quite work as expected.

With `preload`, we have to declare each thing twice:

1. As a link to be preloaded
2. As the actual resource

Check out this [write-up on preload](https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content) for details on how it works.

Reply:
Also, check out [this gist](https://gist.github.com/jlengstorf/4b39f2e5442620f1d26c14d48b580684) for an example of dynamically loading CSS.

Reply:
How about...

```
<link rel="preload" as="style" href="style.css" onload="this.rel='stylesheet'" />
```

Reply:
@gaui I'm not familiar with that approach. What's the goal of the `onload` attribute here?